deploy/config/php-pool.conf

```diff
@@ -18,6 +18,10 @@ pm.max_requests = 500
 pm.max_children = 50
 pm.status_path = /status;
 
+; Log a stack trace to stderr for slow queries.
+request_slowlog_timeout = 8s;
+slowlog = /proc/self/fd/2;
+
```

Comment:
This is the same as on [intranet's php-pool](https://github.com/ministryofjustice/intranet/blob/main/deploy/config/php-pool.conf)